### PR TITLE
[iOS] Open PrivacyPolicy URL Link from AboutView

### PR DESCRIPTION
**Pull request by onakama**
## Issue
- close [[iOS] Open PrivacyPolicy URL Link from AboutView #560](https://github.com/DroidKaigi/conference-app-2022/issues/560)

## Overview (Required)
- Open PrivacyPolicy URL Link from AboutView

## Links
- noting.

## Screenshot

https://user-images.githubusercontent.com/80329420/190916518-fcbaafa6-c6d4-40b7-ba7b-87dd45ee6e20.mov

### CODE REVIEW

File: app-ios/Sources/AboutFeature/AboutView.swift [hunk 1]
```diff
@@ -18,8 +18,21 @@ public struct AboutEnvironment {
     public init() {}
 }
 
-public let aboutReducer = Reducer<AboutState, AboutAction, AboutEnvironment> { _, _, _ in
-    return .none
+public let aboutReducer = Reducer<AboutState, AboutAction, AboutEnvironment> { _, action, _ in
+    @Environment(\.openURL) var openURL
```
Comment (ry-itto) [inline app-ios/Sources/AboutFeature/AboutView.swift:22]: Please define this in environment ( `AboutEnvironment` ) 🙏🏼 
Best is to make it injectable from the outside. But it is okey for now 👍🏼

File: app-ios/Sources/AboutFeature/AboutView.swift [hunk 2]
```diff
@@ -18,8 +18,21 @@ public struct AboutEnvironment {
...
+    switch action {
+    case .openAccess:
+        return .none
+    case .openStaffs:
+        return .none
+    case .openPrivacyPolicy:
+        if let url = URL(string: "https://portal.droidkaigi.jp/about/privacy") {
```
Comment (ry-itto) [inline app-ios/Sources/AboutFeature/AboutView.swift:29]: How about implement static url set struct?

for example:

```swift
struct StaticURLs {
  static let privacyPolicy = URL(string: "https://portal.droidkaigi.jp/about/privacy")!
}
```

**Comment by onakama** [comment]
@ry-itto 
I fixed it.
Could you check it?
(I'm not confident.)

**Comment by ry-itto** [comment]
@onakama 
Thanks!
How about place `StaticURLs` in another file and define to top level of file?
`@Environment` point is okey!

**Comment by onakama** [comment]
@ry-itto 
Could you check it?

**Comment by ry-itto** [comment]
@onakama 
Please fix conflicts 🙏🏼

**Comment by onakama** [comment]
@ry-itto 
Could you check it?

**Review by ry-itto — APPROVED** [review]
Looks good! Thanks! 👍🏼

**Comment by onakama** [comment]
@ry-itto 
thanks!!
It was a good experience.

**Pull request merged**